pkg/platform/kube/functionres/lazy.go

```diff
@@ -261,6 +261,14 @@ func (lc *lazyClient) WaitAvailable(ctx context.Context,
 	var deploymentReady bool
 	var ingressReady bool
 	var timeDeploymentReady time.Time
+	var reasonInitContainersNotReady string
+
+	// readiness flag for init containers
+	var initContainersReady bool
```

Comment:
Again, just for consistency:
```suggestion
	var reasonInitContainersNotDone string

	// readiness flag for init containers
	var initContainersDone bool
```

And replace other places in this function (comments etc)